pkg/js/validator/validate-dsl.ts

```diff
@@ -674,31 +688,38 @@ function modelValidation(
           {},
         );
         if (!hasEntry) {
-          const typeIndex = getTypeLineNumber(typeName, lines);
-          const lineIndex = getRelationLineNumber(relationName, lines, typeIndex);
+          const file = typeDef.metadata?.source_info?.file;
+          const module = typeDef.metadata?.module;
+
+          const typeIndex = getTypeLineNumber(typeName, lines); //team 3, group 7,
+          const lineIndex = getRelationLineNumber(relationName, lines, typeIndex); //viewer 6, viewer 10
           if (loop) {
-            collector.raiseNoEntryPointLoop(relationName, typeName, lineIndex);
+            collector.raiseNoEntryPointLoop(relationName, typeName, { file, module }, lineIndex);
           } else {
-            collector.raiseNoEntryPoint(relationName, typeName, lineIndex);
+            collector.raiseNoEntryPoint(relationName, typeName, { file, module }, lineIndex);
           }
         }
       }
     });
   }
 
-  for (const conditionName in authorizationModel.conditions) {
-    // const condition = authorizationModel.conditions[conditionName];
-    // Ensure that the nested condition name matches
-    // TODO: This does not make sense for the DSL, and is a JSON only error
-    // if (conditionName != condition.name) {
-    //   const conditionIndex = geConditionLineNumber(conditionName, lines);
-    //   collector.raiseDifferentNestedConditionName(conditionIndex, conditionName);
-    // }
-
-    // Ensure that the condition has been used
-    if (!usedConditionNamesSet.has(conditionName)) {
-      const conditionIndex = geConditionLineNumber(conditionName, lines);
-      collector.raiseUnusedCondition(conditionName, conditionIndex);
+  if (authorizationModel.conditions) {
+    for (const [conditionName, condition] of Object.entries(authorizationModel.conditions)) {
+      // const condition = authorizationModel.conditions[conditionName];
```

Comment:
Do we need this commented code?

Reply:
Done in 9cc4c6e9156a7468341b2eb127cdf20736b5bf21 and added a test for `validateJSON`